Diffusion/Pipeline/Pipeline.swift

```diff
@@ -29,15 +29,16 @@ class Pipeline {
         self.pipeline = pipeline
     }
     
-    func generate(prompt: String, scheduler: StableDiffusionScheduler, numInferenceSteps stepCount: Int = 50, seed: UInt32? = nil) throws -> CGImage {
+	func generate(prompt: String, scheduler: StableDiffusionScheduler, numInferenceSteps stepCount: Int = 50, imageCount: Int = 1, safetyOn: Bool = false, seed: UInt32? = nil) throws -> CGImage {
```

Comment:
Can we revert the UI changes and additional options from this PR? They are very useful, but I think it's better to deal with them separately if that's possible.